testsuite/resources/service_mesh/plugin.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: threescale-extension
+objects:
+- apiVersion: extensions.istio.io/v1alpha1
+  kind: WasmPlugin
+  metadata:
+    name: ${NAME}
+    labels:
+      app: ${LABEL}
+  spec:
+    url: oci://${IMAGE}
+    phase: AUTHZ
```

Comment:
If used with the released image, we will also need to add a field containing a pullSecret.

Reply:
Added support for this